### PR TITLE
cypress: added missing cypress test case for glossary feature-part-1

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Added below test case
- Validate POST Api call while creating glossary
- Add tags while creating glossary
- Validate Tags which was added while creating glossary
- Remove tags from glossary

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 1]
```diff
@@ -233,7 +238,29 @@ describe('Glossary page should work properly', () => {
...
+      expect(request.body.mutuallyExclusive).equals(true);
+      expect(request.body.owner).to.have.all.keys('id', 'type');
+      expect(request.body.reviewers).has.length(1);
+      expect(request.body.tags).has.length(1);
+      expect(request.body.tags[0]).to.deep.equal({
+        labelType: 'Manual',
+        state: 'Confirmed',
+        tagFQN: 'PersonalData.Personal',
+        source: 'Tag',
+      });
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:263]: Can't we directly check for object itself in one go?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 2]
```diff
@@ -261,7 +288,24 @@ describe('Glossary page should work properly', () => {
...
+        'reviewers',
+        'tags'
+      );
+      expect(request.body.displayName).equals(NEW_GLOSSARY_1.name);
+      expect(request.body.name).equals(NEW_GLOSSARY_1.name);
+      expect(request.body.description).equals(NEW_GLOSSARY_1.description);
+      expect(request.body.mutuallyExclusive).equals(false);
+      expect(request.body.owner).to.have.all.keys('id', 'type');
+      expect(request.body.reviewers).has.length(0);
+      expect(request.body.tags).has.length(0);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:307]: Same

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:263]: Reviewer and user ID are generated from backend, and everytime its generates new ID and that data we dont have! Please let me know if you have any suggestion!

**Review by chirag-madlani — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/DataConsumerRole.spec.js [hunk 3]
```diff
@@ -253,7 +253,7 @@ describe('DataConsumer Edit policy should work properly', () => {
     }
     cy.get('body').click();
 
-    cy.get('[data-testid="add-new-glossary"]')
+    cy.get('[data-testid="add-glossary"]')
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/DataConsumerRole.spec.js:256]: We have both the buttons, Can you make sure that this change is required one?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 4]
```diff
@@ -11,6 +11,7 @@
  *  limitations under the License.
  */
 
+// / <reference types="Cypress" />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:14]: It won't work

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 5]
```diff
@@ -162,12 +163,9 @@ describe('Glossary page should work properly', () => {
     interceptURL('POST', '/api/v1/glossaries', 'createGlossary');
 
     // check for no data placeholder
-    cy.get('[data-testid="add-new-glossary"]')
-      .should('be.visible')
-      .as('addNewGlossary');
+    cy.get('[data-testid="add-glossary"]').should('be.visible').click();
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:166]: same

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/DataConsumerRole.spec.js (hunk 3 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/DataConsumerRole.spec.js:256]: Yes, with this change we can run cypress for glossaries even if other glossary data already exist, earlier if other gossary data was there it user to fail the test

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js (hunk 4 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:14]: i will fix in my next PR

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**